### PR TITLE
NVMe host name could be unassigned

### PR DESCRIPTION
**Pull request by baoy1**
# Description
When creating an NVMe host without specifying a name, the PowerFlex manager uses HostType:ID as the display name, and we follow the same behavior.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] AT
- [x] UT
image
image
image

### CODE REVIEW

**Comment by baoy1** [comment]
run e2e test

File: powerflex/provider/nvme_host_datasource_test.go [hunk 1]
```diff
@@ -102,5 +136,11 @@ func TestAccDatasourceNvmeHostNegative(t *testing.T) {
 				ExpectError: regexp.MustCompile(`.*Unable to Read Powerflex NVMe Hosts*.`),
 			},
 		},
+		CheckDestroy: func(_ *terraform.State) error {
```
Comment (doriac11) [inline powerflex/provider/nvme_host_datasource_test.go:139]: Do we need this block?
The provider_test method `testAccePreCheck()` https://github.com/dell/terraform-provider-powerflex/blob/main/powerflex/provider/provider_test.go#L387 resets this before each test.

**Review by doriac11 — APPROVED** [review]

**Comment by forrestxia** [comment]
run e2e test

**Comment by baoy1** [comment]
run e2e test

File: powerflex/provider/nvme_host_datasource_test.go (hunk 1 above)
Comment (baoy1) [reply powerflex/provider/nvme_host_datasource_test.go]: Good point. Removed `CheckDestroy`

**Review by RayLiu7 — APPROVED** [review]

**Comment by baoy1** [comment]
run e2e test

**Pull request merged**